input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,395 @@
+    <thead>
+        <tr>
+            <th style="width: 33.33%;">AU Core</th>
+            <th style="width: 33.33%;">International Patient Access 1.0.0</th>
+            <th style="width: 33.33%;">Comment</th>
+        </tr>
+    </thead>
+    <tbody>
         <tr>
-            <td width="93" valign="top">
-                <p>
-                    <a
-                        href="StructureDefinition-au-core-medication.html"
-                    >
-                        AU Core Medication
-                    </a>
-                </p>
-            </td>
-            <td width="81" valign="top">
-                <p>
-                    <a
-                        href="http://hl7.org/fhir/uv/ipa/StructureDefinition/ipa-medication"
-                    >
-                        IPA-Medication
-                    </a>
-                </p>
-            </td>
-            <td width="111" valign="top">
-                <p>
-                    <img src="https://hl7.org/fhir/R4/assets/images/tick.png"/>
-                </p>
-            </td>
-            <td width="68" valign="top">
-                <p>
-                    <a
-                        href="http://hl7.org/fhir/uv/ips/StructureDefinition/Medication-uv-ips"
-                    >
-                        Medication (IPS)
-                    </a>
-                </p>
-            </td>
-            <td width="130" valign="top">
-                <p>
-                    <img src="https://hl7.org/fhir/R4/assets/images/tick.png"/> if
-                    <a
-                        href="http://hl7.org/fhir/uv/ips/StructureDefinition/Quantity-uv-ips"
-                    >
-                        Data Type Profile: Quantity (IPS)
-                    </a>
-                    constraints are met
-                </p>
-            </td>
-            <td width="78" valign="top">
-                <p>
-                    <a
-                        href="http://hl7.org/fhir/us/core/StructureDefinition/us-core-medication"
-                    >
-                        US Core Medication
-                    </a>
-                </p>
-            </td>
-            <td width="137" valign="top">
-                <p>
-                    <img src="https://hl7.org/fhir/R4/assets/images/tick.png"/> noting that US Core extensibly binds to RxNorm which
-                    is not an AU medicines terminology
-                </p>
-            </td>
+            <td style="width: 33.33%;"><a href="StructureDefinition-au-core-medicationrequest.html">AU Core MedicationRequest</a></td>
+            <td style="width: 33.33%;"><a href="https://hl7.org/fhir/uv/ipa/STU1/StructureDefinition-ipa-medicationrequest.html">IPA-MedicationRequest</a></td>
+            <td style="width: 33.33%;">MedicationRequest.requester type differs: IPA prohibits reference to Organization and RelatedPerson resources.</td>
         </tr>
+    </tbody>
```

Comment:
Missing additional profiles section for IPA

Reply:
Added.